README.md

```diff
@@ -84,6 +87,7 @@ text = "Hello World"
 lang = "en"
 speech = Speech(text, lang)
 speech.play()
+speech.save("output.mp3") # Saving the output to an MP3 file
```

Comment:
I'd move that example below the effects block, since saving to a file is incompatible with applying effects.
Also use a dedicated line for the comment.